source/nodejs/lerna.json

```diff
@@ -1,16 +1,15 @@
 {
 	"version": "independent",
 	"useNx": true,
-	"packages": [
```

Comment:
What does this do?

Reply:
pre-`workspaces`, this was how you told `lerna` what directories contained packages it was responsible for... now it lets `npm` do all of that heavy lifting. `npm` itself knows what to build from here:

https://github.com/microsoft/AdaptiveCards/blob/02c2427da2e99f9d60f7d899fddd263888d7ea45/source/nodejs/package.json#L10-L28